sortedm2m/fields.py

```diff
@@ -376,17 +382,19 @@ def get_intermediate_model_to_field(self, klass):
 
     def create_intermediate_model_from_attrs(self, klass, attrs):
         name = self.get_intermediate_model_name(klass)
-        return type(str(name), (models.Model,), attrs)
+        if self.base_class:
+            return type(str(name), (models.Model, self.base_class), attrs)
+        else:
+            return type(str(name), (models.Model,), attrs)
 
     def create_intermediate_model(self, klass):
         # Construct and return the new class.
         from_field_name, from_field = self.get_intermediate_model_from_field(klass)
         to_field_name, to_field = self.get_intermediate_model_to_field(klass)
         sort_value_field_name, sort_value_field = self.get_intermediate_model_sort_value_field(klass)
-
         meta = self.get_intermediate_model_meta_class(
```

Comment:
Using tools like [autopep8](https://pypi.python.org/pypi/autopep8) should help doing all these. Anyways, will fix them soon.